requirements.yaml

```diff
@@ -4,10 +4,11 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - pkgs/main::python=3.6
-  - bioconda::snakemake=5.18.1
+  - pkgs/main::python=3.8
+  - bioconda::snakemake=5.30.1
+  - pkgs/main::conda=4.9.2
+  - conda-forge::mamba=0.7.1
```

Comment:
are we going to add mamba as requirement? Might make sense?

Reply:
why not? we run all our tests with mamba, so it is guaranteed to work!